### ISSUE TITLE
Onboarding

### ISSUE THREAD

**Issue by 197g** [image-rs/organization#1]
# Welcome

To be added to the `image-rs` organization, simply ask nicely in the comments below. We'll send you an invitation asap. We also plan to give each member space in the [blog](https://github.com/image-rs/blog.image-rs.org) when it is setup.

**Comment by kaj** [comment]
Hi!  I'm using `image` in a photo-managing web application (and some experiments).  I also once contributed some code for faster thumbnailing of large images.  Now that [you asked](https://blog.image-rs.org/2019/04/16/image-rs-hello.html), I'm here to show my interest.

**Comment by 197g** [comment]
Hi :wave: I'm glad you found the blog and also welcome (back) under the new organization.

**Comment by JohnTitor** [comment]
@HeroicKatora Hi, I submitted a minor PR to image-rs/image. Could I become a collaborator?

**Comment by 197g** [comment]
@JohnTitor Yes you can, thank you for the short wait. I see you helped with even more style improvements :)

**Comment by Robzz** [comment]
Hey, may I kindly request an invite ? I'm the guy who initially extracted the TIFF code from image into image-tiff, and lurked around providing the occasional review. I don't expect to be the most active person around, but I'll gladly help here and there when I can.

**Comment by 197g** [comment]
@Robzz Your invitation was a pitfall with *a lot* of other people with very similar user names but yes, of course. Welcome back then.

**Comment by chitalian** [comment]
Hi there, I don't have too much experience with this library. I really like what you all do here and I am eager to learn more rust and image processing. I would love to contribute to this repo. Can I be added as a collaborator? I am hoping to generate some more examples within the next few weeks to learn a bit more about the library

**Comment by aschampion** [comment]
Hi, I'd like an invite. I use image (and a few of the format-specific crates directly) heavily for biomedical image processing and have contributions in at least four of the repos here.

**Comment by 197g** [comment]
@aschampion Will be sent swiftly and well deserved. Welcome :)

**Comment by sirxyzzy** [comment]
@HeroicKatora I have an obscure, and possibly quixotic interest in incorporating support for ILBM files into the Rust ecosystem. ILBM is an IFF based Amiga format, also used in some old games for the PC. I have a prototype decoder I hacked up (see https://github.com/sirxyzzy/ilbm), and I am continuing to test and extend it, it is a long way from primetime, but I wondered if there was any interest in my eventually folding this into "image", as a supported format? If so, I would be happy to to do that work.

**Comment by 197g** [comment]
@sirxyzzy Hi :wave: An obscure format itself is no problem. In fact we already support `farbfeld` which is probably unknown to a lot of users. However, being both rare and a prototype is hard to maintain. I'd be glad to review a PR and the crate as an optional feature when you feel it is relatively stable (note: erroring, but not panicking, on some unsupported features would be fine). Feel free to use the `image-rs` crates for some ideas of how to achieve that, e.g. seting up fuzzing or CI, and don't hesistate to ask in a new issue if anything is unclear or you've made progress.
I've sent a collaborator invite. If you'd like you could make blog posts on major progress points or when you resolve some issue on the way that others might find instructive. In any case, have fun coding :)

**Comment by sirxyzzy** [comment]
@HeroicKatora Thanks, invitation accepted. Yes, I realize I have a way to go, and funny you mentioned panics as I spent a few hours today removing pretty much *all* potential sources of panics, mainly by no longer using the "bytes" crate (this also speeds things up a bit) I totally agree that my code is not ready to be linked in (yet), I have a fair bit of work to go. I was running some tests today with close to 4000 different sample files and assorted other non image formats, good news, no panics, but too many fails to be happy. ILBM is a notoriously flexible format so there are quite a few cases to deal with.

**Comment by nabijaczleweli** [comment]
hey, farbfeld isn't *that* obscure!

**Comment by aarestad** [comment]
Hello! I noticed that `webp` _de_ coding is supported, but not _en_ coding. While I do not have any appreciable experience writing encoders, I was interested in maybe tackling this. Perhaps we could use these bindings to `libwebp`? https://lib.rs/crates/libwebp To that end, I'd like to request an invite to set up an issue and eventually create a PR for it :)

Edit: I see there is an (old) issue already! https://github.com/image-rs/image/issues/582 If someone else is not already doing this, perhaps I can.

Edit2: I see this is out there too: https://lib.rs/crates/libwebp-image - so, never mind - I guess I don't want to duplicate someone's work. ><

**Comment by rbs228** [comment]
hello @HeroicKatora I would like to help.

**Comment by ripytide** [comment]
I have a free summer which I'm intending to dedicate to `imageproc` and all the repos it depends on, including `image`. I'd like to triage all the open but stagnant PRs in the `image` repo.

**Comment by fintelia** [comment]
@ripytide That would be appreciated. I looked though some of the open PRs and it looked like they were mostly either abandoned by their author or stalled out because we didn't reach agreement that the relevant feature should be added

**Comment by fintelia** [comment]
We're no longer using the GitHub teams to coordinate contributors / organization members, so I'm going to go ahead and close this. If you are interested in contributing, please directly open issues or PRs against the relevant repository
